pkg/manager/member/template.go

```diff
@@ -52,7 +52,7 @@ fi
 POD_NAME=${POD_NAME:-$HOSTNAME}{{ if .FormatClusterDomain }}
 pd_url="{{ .Path }}"
 encoded_domain_url=$(echo $pd_url | base64 | tr "\n" " " | sed "s/ //g")
-discovery_url="${CLUSTER_NAME}-discovery.${NAMESPACE}.svc{{ .FormatClusterDomain }}:10261"
+discovery_url="${CLUSTER_NAME}-discovery.${NAMESPACE}.svc:10261"
```

Comment:
```suggestion
discovery_url="${CLUSTER_NAME}-discovery.${NAMESPACE}:10261"
```

Reply:
Just keep the `svc` for PD as it's newly added for the other components and they are protected by `{{ if .FormatClusterDomain }}` so the changes will not affect the current clusters without enabling this feature.

Reply:
thanks for the review, updated